store/actions/threads.js

```diff
@@ -0,0 +1,145 @@
+// Module imp
+import Cookies from 'js-cookie'
+import 'isomorphic-fetch'
+
+
+
+
+
+// Component imports
+import actionTypes from '../actionTypes'
+
+
+
+export const getForumThreads = page => async dispatch => {
+  dispatch({ type: actionTypes.GET_FORUM_THREADS })
+
+  let response = null
+  let success = false
+
+  try {
+    const accessToken = Cookies.get('accessToken')
+
+    response = await fetch(`/api/threads${page ? `?page=${page}` : ''}`, {
```

Comment:
Use the `apiService` instead of using `fetch` directly.

Reply:
See comment on #133 about this.